File: DSF/factorial.c
//write a program to find the factorial of a number
#include <stdio.h>
int main()
{
    int i, n, fact = 1;
    printf("Enter the number:");
    scanf("%d", &n);
    for (i = 1; i <= n; i++)
    {
        fact = fact * i;
    }
    printf("Factorial of %d is %d", n, fact);
    return 0;
}

File: Java/java/Experiment 1/factorial.java
//factorial of a number
import java.util.*;
public class factorial {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter a number:");
        int num = in.nextInt();
        int fact = 1;
        for (int i = 1; i <= num; i++) {
            fact = fact * i;
        }
        System.out.println("Factorial of " + num + " is: " + fact);
    }
}